root.go

```diff
@@ -1,15 +1,32 @@
 package brimcap
 
 import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"io/fs"
+	"net"
 	"os"
 	"path/filepath"
 
 	"github.com/brimdata/brimcap/pcap"
-	"github.com/brimdata/zed/pkg/fs"
+	"github.com/brimdata/brimcap/pcap/pcapio"
+	fse "github.com/brimdata/zed/pkg/fs"
```

Comment:
Nit: As a reader, I find renamed packages to be easier to follow if renamed mechanically by concatenating trailing path components, like `pks/fs` here.